### DISCUSSION TITLE
Update message

### DISCUSSION THREAD

**Discussion by ashy516** [monicahq/monica#5434] [Q&A]
Greetings,
I just installed Monica on my TrueNAS Jail server and it was working fine on version 2.0 (I think) and started populating it with information. However, since I was behind by several updates, I went ahead and updated it. However, it returned with this page below. I'm not an expert on Linux. What does this mean and what do I need to do to restore the functionalities back? I would hate to delete all the work I have done.

Thank you.

![Message](https://user-images.githubusercontent.com/9933473/129751932-c42ffb61-6a99-48e4-a051-0356ce5f82fb.png)

**Comment by asbiin** [comment]
It seems you did not properly setup you http webserver.
Be sure to setup your apache/nginx and php (fpm?).

**Reply by ashy516** [reply]
Thanks for your response. Not much of a Linux guru, would you have any suggestion how to properly configure apache/nginx and php to make this work? This is a very impressive app for personal use and I would like to revive it. Thanks.
